### PR TITLE
Fix #5089 and don't auto-load Registries

### LINKED ISSUE #5080: player nametags text plugins are not displayed correctly on geyser side (bug)
### Describe the bug

basically using some plugins to change their player nametags with custom text are not displayed correctly on bedrock side

### To Reproduce

install unlimited tags plugin
and just see the nametag for both players, the bedrock side is wrong position

### Expected behaviour

correct the position and the name duplication on bedrock

### Screenshots / Videos

bug

### Server Version and Plugins

papermc build 120
unlimited tags lastest version
minecrft 1.21.1

### Geyser Dump

_No response_

### Geyser Version

1.21.1

### Minecraft: Bedrock Edition Device/Version

1.21.1

### Additional Context

_No response_

### LINKED ISSUE #5089: NPC does not display the correct name
### Describe the bug

The NPC does not display the correct name after updating Geyser to commit ef4acb121f0f8746fa18a27f6acb1fc032ec7b2b. This might be related to issues #5085 and #5080. The last commit where this bug did not occur was c656e41.

### To Reproduce

1. Update Geyser to the latest version.
2. Install the [ZNPCsPlus](https://www.spigotmc.org/resources/znpcsplus.109380/) plugin on Paper.
3. Run `/npc create 1 player`.
4. Run `/npc holo add 1 tesss`.

### Expected behaviour

image

### Screenshots / Videos

image

### Server Version and Plugins

_No response_

### Geyser Dump

_No response_

### Geyser Version

2.4.3-b685 (git-master-ef4acb1) (Java: 1.21, Bedrock: 1.20.80/1.20.81 - 1.21.30/1.21.31)

### Minecraft: Bedrock Edition Device/Version

1.21.31 Windows, 1.21.20 Android

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Tim203**
Also fixes #5080

### CODE REVIEW

File: core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    /**
+     * A registry containing all the Java packet translators.
+     */
+    PacketTranslatorRegistry<Packet> javaPacketTranslators();
+
+    /**
+     * A registry containing all Java items ordered by their network ID.
+     */
+    ListRegistry<Item> javaItems();
+    SimpleMappedRegistry<String, Item> javaItemIdentifiers();
```
Comment (onebeastchris) [inline core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java:106]: missing javadocs, method spacing

File: core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+public interface CommonRegistries {
+    /**
+     * A registry holding all the providers.
+     * This has to be initialized first to allow extensions to access providers during other registry events.
+     */
+    SimpleMappedRegistry<Class<?>, ProviderSupplier> providers();
+
+    /**
+     * A registry holding a CompoundTag of the known entity identifiers.
```
Comment (onebeastchris) [inline core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java:62]:
```suggestion
     * A registry holding a NbtMap of the known entity identifiers.
```

**Review by onebeastchris — COMMENTED** [review]
Moving the java imports would also probably be better suited for a different PR, otherwise future Pars touching these classes will create unnecessary diff

File: core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java (hunk 1 above)
Comment (Tim203) [reply core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java:106]: The current Registries is also missing a javadoc, and the method spacing is done for easier reading (clustering similar registries)

File: core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java (hunk 2 above)
Comment (Tim203) [reply core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java:62]: The javadocs are just copied from the Registries. I'm fine with changing it though

File: core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java (hunk 1 above)
Comment (onebeastchris) [reply core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java:106]: seems like a good opportunity to add a javadoc, just for consistency :p

File: core/src/main/java/org/geysermc/geyser/entity/type/living/ArmorStandEntity.java [hunk 3]
```diff
@@ -25,8 +25,11 @@
 
 package org.geysermc.geyser.entity.type.living;
 
+import java.util.Optional;
+import java.util.UUID;
```
Comment (onebeastchris) [inline core/src/main/java/org/geysermc/geyser/entity/type/living/ArmorStandEntity.java:29]: please revert

File: core/src/main/java/org/geysermc/geyser/translator/protocol/java/entity/player/JavaPlayerInfoUpdateTranslator.java [hunk 4]
```diff
@@ -25,7 +25,10 @@
 
 package org.geysermc.geyser.translator.protocol.java.entity.player;
 
-import org.geysermc.mcprotocollib.auth.GameProfile;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
 import org.checkerframework.checker.nullness.qual.Nullable;
```
Comment (onebeastchris) [inline core/src/main/java/org/geysermc/geyser/translator/protocol/java/entity/player/JavaPlayerInfoUpdateTranslator.java:32]: please revert

File: core/src/test/java/org/geysermc/geyser/scoreboard/network/ScoreboardIssueTests.java [hunk 5]
```diff
@@ -25,23 +25,57 @@
...
+import static org.geysermc.geyser.scoreboard.network.util.AssertUtils.assertNextPacketMatch;
 import static org.geysermc.geyser.scoreboard.network.util.AssertUtils.assertNextPacketType;
+import static org.geysermc.geyser.scoreboard.network.util.AssertUtils.assertNoNextPacket;
 import static org.geysermc.geyser.scoreboard.network.util.GeyserMockContextScoreboard.mockContextScoreboard;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+import java.util.EnumSet;
+import java.util.Optional;
+import java.util.UUID;
```
Comment (onebeastchris) [inline core/src/test/java/org/geysermc/geyser/scoreboard/network/ScoreboardIssueTests.java:38]: imports

File: core/src/test/java/org/geysermc/geyser/scoreboard/network/ScoreboardIssueTests.java [hunk 6]
```diff
@@ -90,4 +124,137 @@ void entityWithoutType() {
...
+            context.translate(
+                addEntityTranslator,
+                new ClientboundAddEntityPacket(1298, hologramUuid, EntityType.ARMOR_STAND, 6, 5, 4, 3, 2, 1));
+
+            assertNextPacketMatch(context, AddEntityPacket.class, packet -> {
+                assertEquals(4, packet.getRuntimeEntityId());
+                assertEquals("minecraft:armor_stand", packet.getIdentifier());
+            });
+
+            // invisible, custom name, custom name visible
```
Comment (onebeastchris) [inline core/src/test/java/org/geysermc/geyser/scoreboard/network/ScoreboardIssueTests.java:216]: Sorry, what does this mean? is that just a visibility change from invisible -> visible custom name?

File: core/src/main/java/org/geysermc/geyser/registry/loader/RegistryLoaderHolder.java [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/Geyser
+ */
+
+package org.geysermc.geyser.registry.loader;
+
+public record RegistryLoaderHolder<I, M>(I input, RegistryLoader<I, M> registryLoader) {
```
Comment (onebeastchris) [inline core/src/main/java/org/geysermc/geyser/registry/loader/RegistryLoaderHolder.java:28]: javadoc would be neat, just for future reference/help for new contributors

**Review by onebeastchris — APPROVED** [review]
Other than these, looks good

File: core/src/test/java/org/geysermc/geyser/scoreboard/network/ScoreboardIssueTests.java (hunk 6 above)
Comment (Tim203) [reply core/src/test/java/org/geysermc/geyser/scoreboard/network/ScoreboardIssueTests.java:216]: It is a comment for the Entity metadata I send. Index 0 with 0x20 is the flag for invisibility. Index 2 stands for the custom name. Index 3 stands for custom name visible.

**Review by Camotoy — APPROVED** [review]

**Pull request merged**